test/testflows/clickhouse_backup/tests/cloud_storage.py

```diff
@@ -194,8 +194,8 @@ def s3_aws(self):
 def gcs(self):
     """Test that an existing backup can be uploaded to GCS bucket.
     """
-    if not os.environ.get('QA_GCS_CRED_JSON'):
-        skip("QA_GCS_CRED_JSON environment not found, test skipped")
+    if not (os.environ.get('QA_GCS_CRED_JSON') or os.environ.get('QA_GCS_CRED_JSON_ENCODED')):
+        skip("QA_GCS_CRED_JSON/QA_GCS_CRED_JSON_ENCODED environment not found, test skipped")
 
     test_storage_outline(storage_type="gcs")
```

Comment:
did you look inside test_storage_outline? you shall handle `QA_GCS_CRED_JSON_ENCODED`

Reply:
yes, I've updated code

Reply:
but test still not coverage `QA_GCS_CRED_JSON_ENCODED`

cause it not handle `inside test_storage_outline`